src/include/uapi/drm_local/amdxdna_accel.h

```diff
@@ -163,27 +163,51 @@ struct amdxdna_ctx_param_config_cu {
 	struct amdxdna_cu_config cu_configs[];
 };
 
+/**
+ * struct amdxdna_drm_ctx_config_metadata - metadata to configure context.
+ * @magic_no: CERT magic number
+ * @major: CERT major number
+ * @minor: CERT minor number
+ * @debug_buf_flag: Specifies the type of buffer.
+ * @num_cols: total coulmns count
+ * @col_paddr: physical address of debug buffer assigned to this col
+ * @col_size: size of the debug buffer
+ * @pad: Structure padding.
+ */
+struct amdxdna_drm_ctx_config_metadata {
+	__u32 magic_no;
+	__u8 major;
+	__u8 minor;
+#define DRM_AMDXDNA_CTX_CONFIG_CU               0
+#define DRM_AMDXDNA_CTX_ASSIGN_DBG_BUF          1
+#define DRM_AMDXDNA_CTX_REMOVE_DBG_BUF          2
+#define DRM_AMDXDNA_CTX_ASSIGN_DTRACE_BUF       3
```

Comment:
We shouldnt be introducing any new defines. Everything should be handled under DBG_BUF only.